rayon-threadlimit/src/lib.rs

```diff
@@ -16,3 +16,9 @@ lazy_static! {
 pub fn get_thread_count() -> usize {
     *MAX_RAYON_THREADS
 }
+
+// Only used in legacy code.
+// Use get_thread_count instead in all new code.
+pub fn get_max_thread_count() -> usize {
+    get_thread_count().saturating_mul(2)
```

Comment:
This is still doubling the envvar-specified values.  I think we need a separate global, unfortunately

Reply:
The goal here is to have same number of threads as in the old code.
So the only change in this commit will be `thread_local -> lazy_static`.
The old code was initializing twice as many threads as the envvar, so `*2` here to keep it the same.

I think the best would be to have separate commits if we want to change number of threads; since that is a change which requires much more testing.

Reply:
@t-nelson are you ok with this? this is effectively the same as the current master.

Reply:
yep, ship it.  One day I'll remember that the multiplication comes from threads, not math